Source/Core/VideoCommon/DriverDetails.cpp

```diff
@@ -58,6 +58,7 @@ namespace DriverDetails
 		{OS_WINDOWS,VENDOR_NVIDIA,   DRIVER_NVIDIA,       -1, BUG_BROKENUNSYNCMAPPING, -1.0, -1.0, true},
 		{OS_LINUX,  VENDOR_NVIDIA,   DRIVER_NVIDIA,       -1, BUG_BROKENUNSYNCMAPPING, -1.0, -1.0, true},
 		{OS_WINDOWS,VENDOR_INTEL,    DRIVER_INTEL,        -1, BUG_INTELBROKENBUFFERSTORAGE, 101810.3907, 101810.3960, true},
+		{OS_ALL,    VENDOR_MESA,     DRIVER_I965,         -1, BUG_BROKENIMAGELOADSTORE,-1.0, -1.0, true},
```

Comment:
i915 vs i965 is a very common issue.

The linux DRM driver i915 is for all intel GPUs, up to skylake. There is no i965 DRM driver.

The mesa i915 driver is for the very old fixed function GPUs, we don't support them. The mesa driver i965 is for the newer ones, afaik somewhere about core2 and newer.